src/expr/src/vector_op/arithmetic_op.rs

```diff
@@ -327,6 +327,63 @@ where
     r.mul_float(l).ok_or(ExprError::NumericOutOfRange)
 }
 
+// TODO: find or create a trait that contains sqrt method for numeric types and refactor below with
+// generics.
```

Comment:
I guess this method is already contained in `num_traits::real::Real` which is already imported. Would you like to try it now?

Reply:
thank you, i fixed this with `num_traits::Float` as it contains a bit more methods such as checking against nan or infinity. Also I found that i do not need to implement for integers (i64, i32 ..etc.) which simplifies the new commit.